src/templates/footer.html

```diff
@@ -32,7 +32,7 @@
   <div class="footer-content">
     <h1 class="footzilla"><a href="https://www.mozilla.org/" target="_blank">mozilla</a></h1>
     <p>
-      {{ _('Except where otherwise <a href="{legal_url}" target="_blank">noted</a>, content on this site is licensed under the <a href="{cc_url}" target="_blank">Creative Commons Attribution Share-Alike License v3.0</a> or any later version.',
+      {{ _('<span style="color:#717268">Except where otherwise <a href="{legal_url}" target="_blank">noted</a>, content on this site is licensed under the <a href="{cc_url}" target="_blank">Creative Commons Attribution Share-Alike License v3.0</a> or any later version.</span>',
```

Comment:
inline styles are bad since they add a lot of specificity. preferably we add a class on the ```p``` and add the color thru a stylesheet, using one of our Stylus color variables.